internal/exec/stages/mount/mount.go

```diff
@@ -133,6 +135,9 @@ func (s stage) mountFs(fs types.Filesystem) error {
 	}
 
 	if _, err := os.Stat(path); err != nil && os.IsNotExist(err) {
+		if _, err := util.PathExists(path); err != nil {
```

Comment:
The previous line verifies that the path does _not_ exist, so this condition will never be true.  Did you test this?

Reply:
I thought since IsNotExist had err being passed to it it wasn't checking paths, thanks for clearing that up!

I wasn't too sure how to test it at the time but i think i get the idea now i ran ./test on the most recent push and everything passed